safe/report/extractors/action_notes.py

```diff
@@ -255,7 +255,6 @@ def action_checklist_report_pdf_extractor(impact_report, component_metadata):
     # - Substitution maps
     # - Element settings, such as icon for picture file or image source
 
-    _ = component_metadata
```

Comment:
If you still need that, to mark that the var is not used you need:
`_ = component_metadata  # NOQA` and then everything is fine. I agree it could be nice automatically a `_` variable.

Reply:
Yeah. I will let it since I just pick it up when I open this file.

I forget to add NOQA previously :)

Thanks

Reply:
yep, because pycharm will now complain too ;-)